packages/arb-bridge-eth/test/arbRollup.spec.ts

```diff
@@ -405,4 +436,172 @@ describe('ArbRollup', () => {
   it('returns stake to staker', async function () {
     await rollup.returnOldDeposit(await accounts[2].getAddress())
   })
+
+  it('should pause the contracts then resume', async function () {
+    const prevIsPaused = await rollup.rollup.paused()
+    expect(prevIsPaused).to.equal(false)
+
+    await rollupAdmin.pause()
+
+    const postIsPaused = await rollup.rollup.paused()
+    expect(postIsPaused).to.equal(true)
+
+    await expect(
+      rollup
+        .connect(accounts[2])
+        .addToDeposit(await accounts[2].getAddress(), { value: 5 })
+    ).to.be.revertedWith('Pausable: paused')
+
+    await rollupAdmin.resume()
+  })
+
+  it('should allow admin to truncate nodes', async function () {
```

Comment:
This test is way broader than the title suggests. Should probably have a more general name. If there's time it'd also be good to consider ways to break this up into smaller chunks